releases/release-1.29/release-notes/maps/pr-120330-map.yaml

```diff
@@ -1,3 +1,3 @@
 pr: 120330
 releasenote:
-  text: Retry `NodeStageVolume` calls if the CSI node driver is not running.
+  text: Retried `NodeStageVolume` calls if the CSI node driver is not running.
```

Comment:
I feel it can rather be - 

text: '`NodeStageVolume` calls will now retry if the `CSI` node driver is not running.'